RecoLocalMuon/DTSegment/python/DTMeantimerPatternReco2DAlgo_LinearDriftFromDB_CosmicData_cfi.py

```diff
@@ -7,16 +7,17 @@
 # The linear DriftFromDB algos is used.
```

Comment:
technically, these are not module initializer files
but none of it has been enforced by the config file model

https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideAboutPythonConfigFile#Including_Standard_Module_defini